### ISSUE TITLE
[Accessibility][JAWS][Activity Performance Report]: Screen reader does not announces the tooltip message

### ISSUE THREAD

**Issue by nikitaorliak-cengage** [cengage/react-magma#2202]
[A11Y-4105](https://jira.cengage.com/browse/A11Y-4105)

Environment:
Windows 11/Chrome 137/JAWS 2025 
Windows 11/Chrome 137/NVDA 2025
Mac15/ Safari/ Voiceover 

Steps to reproduce:
1. Open the URL – https://react-magma.cengage.com/version/4.10.1/api/tooltip/
2. Turn on the screen reader (JAWS/NVDA/VoiceOver)
3. Go to the `On non-interactive element` section
4. Move focus to the Tooltip
5. Observe that the aria-label is announced when `Tooltip's` Content is not

Actual Result:
The screen reader does not announce the content, even though it is visible to sighted users.

Expected Result:
Screen reader should read both button label and tooltip message

WCAG Guideline:
1.3.1 Info & relationship

User Impact:
Screen reader users will miss important contextual information. Can lead to incomplete understanding, incorrect decisions, or missed actions. Creates an unequal experience compared to sighted users.

Recommendation:
Add a Tooltip Container with role="tooltip" and Link the Button Using aria-describedby

---------------------------------------------------------------------------------------------

We had this conversation before, and we recommend using the same string for Tooltip's content and for aria-label. In case if they are different, we can see the screenreader only announces aria-label.

**Comment by silvalaura** [comment]
Left a comment on JIRA ticket-- this work needs to be done on MT's side to update their aria-label
